devel/ci/bodhi_ci/unit.py

```diff
@@ -47,13 +48,13 @@ def __init__(self, *args, **kwargs):
         modules = " ".join(self.options["modules"])
 
         test_command = (
-            # Run setup.py develop in each submodule
-            f'for submodule in {modules}; do '
+            # Run setup.py develop in all 3 submodules
+            f'for submodule in {" ".join(MODULES)}; do '
             'cd $submodule; /usr/bin/python3 setup.py develop; cd ..; done; '
             # Run the tests in each submodule
             f'for submodule in {modules}; do '
             'mkdir -p /results/$submodule; '
-            f'cd $submodule; /usr/bin/python3 -m pytest {pytest_flags} || '
+            f'cd $submodule; /usr/bin/python3 -m pytest {pytest_flags}; exit $? || '
```

Comment:
Nope, that can't work, it'll exit after the first module every time

Reply:
But it's the right place to fix it! :-) I have fixed it in devel, I'll push it and you can just rebase.

Reply:
done, removed my last commit and rebased with your fix